types/components/FormCheckInput.d.ts

```diff
@@ -9,6 +9,7 @@ export interface FormCheckInputProps {
   isValid?: boolean;
   isInvalid?: boolean;
   innerRef?: React.LegacyRef<this>;
+  as?: React.ElementType;
 }
 
 declare class FormCheckInput extends BsPrefixComponent<
```

Comment:
copy the typing from above with `As extends React.ElementType = 'input'`

Reply:
I'm not totally clear. If I assume that means FormControl.d.ts was correct as is, I guess that means I shouldn't have `as` listed in the props. About to push an attempt...